README.md

```diff
@@ -1,5 +1,5 @@
 # How to launch
-1. Clone repo
+1. Clone repo 'cd ~/Documents/Projects/express-api-swagger'
```

Comment:
This is not full instruction on how to clone the repo. Also, this path is your local only, you should avoid local paths in common instructions